flipperdevices/flipperzero-firmware
# [FL-1929, FL-2164] IR App migrate to FFF

# What's new

- Migrate to FFF (works veeeeeeeeery slow)
- Update universal TV library
- Refactoring IR app
- Fix FFF values counting
- Fix IR error: sending only common frequency (38k) and duty cycle(0.33) in spite of file values
- [FL-2164] hide unimplemented universal menus
- Fix IR TX worker race

# Verification 

- Upload new TV library and run brute force
- Run IR app, save some IR signals, open SD card with PC and check it has FFF

# Checklist (do not modify)



[FL-2164]: https://flipperzero.atlassian.net/browse/FL-2164?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
